contracts/facade/factories/CollateralFactory.sol

```diff
@@ -2,14 +2,18 @@
 pragma solidity 0.8.19;
 
 import "../../plugins/assets/DemurrageCollateral.sol";
+import "../../plugins/assets/UnpricedCollateral.sol";
 
 /**
- * @title DemurrageCollateralFactory
+ * @title CollateralFactory
  */
-contract DemurrageCollateralFactory {
+contract CollateralFactory {
     event DemurrageCollateralDeployed(address indexed collateral);
+    event UnpricedCollateralDeployed(address indexed collateral);
 
-    mapping(address coll => uint192 feePerSecond) public demurrageDeployments;
+    mapping(address coll => uint192 feePerSecond) public demurrageDeployments; // not unique by erc20
```

Comment:
`address erc20` => `uint256/uint192 fee` => `address collateral` is much better to query and track.

Reply:
there is not just 1 collateral address possible, per fee tier

Reply:
Sorry I meant for unpriced demurrage collateral. There aren't any specific parameters for those, I don't think right?

Reply:
that's 2 lines below and works the way you want it to

Reply:
well it's saving by address of the deployed collateral. maybe this comment should go in the other pr, but if we save a mapping like akshat suggested (and error on duplicates), then the fe can easily query the contract based on the erc20 addy. otherwise we're requiring a subgraph dependency